bugbug/tools/code_review.py

```diff
@@ -968,13 +988,23 @@ def run(self, patch: Patch) -> list[InlineComment] | None:
 
         memory.save_context(
             {
-                "input": "You are an expert reviewer for the Mozilla Firefox source code, with experience on source code reviews."
+                "input": "You are an expert reviewer for"
+                + (f" in {TARGET_SOFTWARE}" if TARGET_SOFTWARE is not None else "")
+                + " source code, with experience on source code reviews."
+            },
+            {
+                "output": "Sure, I'm aware of source code practices"
+                + (
+                    f" in {TARGET_SOFTWARE}"
+                    if TARGET_SOFTWARE is not None
+                    else " in the development community"
+                )
+                + "."
             },
-            {"output": "Sure, I'm aware of source code practices in Firefox."},
         )
         memory.save_context(
             {
-                "input": 'Please, analyze the code provided and report a summarization about the new changes; for that, focus on the code added represented by lines that start with "+". '
+                "input": 'Please, analyze the code provided and report a summarization about the new changes; for that, focus on the code added represented by lines that start with "+". \n'
```

Comment:
nit: remove whitespace ```suggestion
                "input": 'Please, analyze the code provided and report a summarization about the new changes; for that, focus on the code added represented by lines that start with "+".\n'
```

Reply:
done